service_storagefacility/src/main/scala/no/uio/musit/microservice/storagefacility/resource/StorageNodeResource.scala

```diff
@@ -336,5 +336,26 @@ final class StorageNodeResource @Inject() (
       Future.successful(BadRequest(Json.obj("message" -> s"Unknown museum $mid")))
     }
   }
+
+  def currentObjectLocation(mid: Int, oid: Long) = Action.async { implicit request =>
+    Museum.fromMuseumId(mid).map { museum =>
+      service.getCurrentObjectLocation(mid, oid).map {
+        case MusitSuccess(optCurrLoc) =>
+          optCurrLoc.map { CurrLoc =>
+            Ok(Json.toJson(CurrLoc))
+          }.getOrElse {
+            NotFound(Json.obj("message" -> s"Could not find objectId $oid"))
```

Comment:
It's helpful to provide the museumId in the response message, since we know it.

```scala
NotFound(Json.obj("message" -> s"Could not find objectId $oid in museum $museumId"))
```